src/main/java/io/beanmapper/config/BeanMapperBuilder.java

```diff
@@ -119,13 +120,13 @@ public BeanMapperBuilder setBeanUnproxy(BeanUnproxy beanUnproxy) {
 
     public BeanMapperBuilder downsizeSource(List<String> includeFields) {
         this.configuration.setApplyStrictMappingConvention(false);
-        this.configuration.downsizeSource(includeFields);
+        this.configuration.downsizeSource(includeFields != null ? includeFields : new ArrayList<>());
         return this;
     }
 
     public BeanMapperBuilder downsizeTarget(List<String> includeFields) {
         this.configuration.setApplyStrictMappingConvention(false);
-        this.configuration.downsizeTarget(includeFields);
+        this.configuration.downsizeTarget(includeFields != null ? includeFields : new ArrayList<>());
```

Comment:
Is er reden waarom je hier new ArrayList gebruikt maar voor de rest steeds Collections.emptyList();?

Reply:
Nee, zo te zien niet. Zal dat even netjes wijzigen naar Collections.emptyList()